commit 1bfd76aed8eccc841221ba9787411703e40c00b4
Author: Tim Foster <tim.foster@joyent.com>
Date:   2019-06-26T18:09:27+01:00 (3 months ago)
    
    TOOLS-2283 move manta and smartos jr repos.json out of triton-dev
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/repos.json b/tools/jr-manifest.json
similarity index 100%
rename from repos.json
rename to tools/jr-manifest.json
